core/client/fs/src/main/java/alluxio/client/block/stream/GrpcBlockingStream.java

```diff
@@ -145,14 +148,56 @@ public void send(ReqT request) throws IOException {
    * @throws IOException if any error occurs
    */
   public ResT receive(long timeoutMs) throws IOException {
+    return retryReceive(timeoutMs, null);
```

Comment:
is there any point in having the `receive` method if it directly calls `retryReceive`? Or rather, could the implementation of `retryReceive` simply be `receive`?